Makefile

```diff
@@ -229,8 +231,7 @@ check-licenses: ## Checks license status.
 check-prereqs: ## Checks prerequisite software status.
 	./scripts/prereq-check.sh
 
-# TODO: remove govet and gofmt checks once golangci-lint is being enforced.
-check-style: govet gofmt check-licenses check-plugin-golint ## Runs govet and gofmt against all packages and also ensures plugin package golint compliant
+check-style: golangci-lint checker check-licenses check-plugin-golint ## Runs golangci against all packages and also ensures plugin package golint compliant
```

Comment:
We can remove `gofmt` and `govet` now, right?

Reply:
Yeah, it _is_ removed. Did I misunderstand something ?

Reply:
I mean removing the Makefile targets completely.